src/groupREADME.md

```diff
@@ -0,0 +1,76 @@
+##Abstract:
+Many people have routine eating pallets and  what's cooking good looking offers many unique recipes as well as cooking instructions to advance a persons culinary experience. 
+
+What's Cooking Good Looking: is a web application that provides a user the abilities of:  being able to search a recipe, save a recipe, delete a saved recipe,filter all recipes,filter all saved recipes, search by tags, and see specifications on any app compiled recipe. 
+
+##Installation Instructions:
+1. start at https://github.com/daniabee/whats-cookin-good-lookin
+2. click CODE
+3. Copy SSH 
+4. Go to computer terminal type: git clone <'filename'> 
+5. cd into directory 
+6. run 'npm install'
+7. run 'npm start'
+8. should be given a server page 'http://localhost:8080/'
+9. open server in browser.  
+
+
+##Preview of App:
```

Comment:
I will get the preview of the app once Dani is done with the CSS!

Reply:
No rush I have a space set just for it!